website/src/stylesheets/_gallery.scss

```diff
@@ -12,7 +12,7 @@
 }
 .toc {
   min-width: 240px;
-  max-width: 320px;
+  max-width: 360px;
```

Comment:
Let's leave this out for now. We will drop the (experimental) tag from the title in the next release.